contributors/command-help.md

```diff
@@ -1,117 +0,0 @@
-# SRE-BOT Command Help
-
-## Continuous Integration(CI) Commands
-
-Supported Repositories:
-
-* [tidb](https://github.com/pingcap/tidb)
-* [tikv](https://github.com/tikv/tikv)
-* [pd](https://github.com/pingcap/pd)
-* [tispark](https://github.com/pingcap/tispark)
-* [tidb-binlog](https://github.com/pingcap/tidb-binlog)
-* [tidb-operator](https://github.com/pingcap/tidb-operator)
-
-### `/rebuild`
-
-#### Description
-
-Build the binary based on the PR for testing. It also reruns the following
-continuous integration(CI) test cases:
-
-* `unit-test`
-* `build`
-* `build_check_race`
-* `check_dev`
-* `check_dev_2`
-
-### `/run-all-tests`
-
-#### Description
-
-Triggers all the CI test cases. Here is the list of all the CI test cases:
-
-* `integration-common-test`
-* `integration-compatibility-test`
-* `integration-ddl-test`
-
-For the [tidb](https://github.com/pingcap/tidb) repository, there are five more
-test cases in the CI:
-
-* `unit-test`
-* `common-test`
-* `mybatis-test`
-* `sqllogic-test-1`
-* `sqllogic-test-2`
-
-#### Parameters
-
-```
-# specify which tidb to use
-tidb=<branch>|<pr/$num>
-
-# specify which tikv to use
-tikv=<branch>|<pr/$num>
-
-# specify which pd to use
-pd=<branch>|<pr/$num>
-
-# specify which tidb-test to use
-tidb-test=<branch>|<pr/$num>
-
-# specify which tidb-private-test to use
-tidb-private-test=<branch>|<pr/$num>
-```
-
-#### Examples
-
-```
-/run-all-tests tidb-test=pr/666
```

Comment:
Similar commands still seem to be used, maybe we should correct it instead of deleting it.

https://github.com/pingcap/tidb/pull/26876#issuecomment-892390347

Reply:
They are not always true. I'll close this PR due to https://github.com/pingcap/community/pull/514#discussion_r684579657. But if not, I tend to delete this one and create a follow-ups PR to describe true CI commands. Or on our dev guide.